### PR TITLE
style : alert.css

### PR DESCRIPTION
**Pull request by Bibin-Antony**
made some minor corrections in the alert component.

### CODE REVIEW

**Comment by Bibin-Antony** [comment]
image

File: Container/CSS/style.css [hunk 1]
```diff
@@ -45,31 +52,32 @@
     gap: 1.2rem;
 }
 
-.nav-bar a{
-    padding: 0.4rem 1rem ;
+.nav-bar a {
```
Comment (ShoaibTamboli) [inline Container/CSS/style.css:55]: just beautify the file.i.e. remove extra spaces.

**Review by Creator-singh — APPROVED** [review]
good to go ahead

**Comment by Bibin-Antony** [comment]
image

File: Container/CSS/style.css (hunk 1 above)
Comment (Bibin-Antony) [reply Container/CSS/style.css:55]: resolved, thanks!

**Pull request merged**